### ISSUE TITLE
NullPointerException - CleverTap Plugin 1.2.3

### ISSUE THREAD

**Issue by tlworkssoftware** [CleverTap/clevertap-flutter#61]
I've been using the clevertap plugin for Flutter version 2.2.3 to use Native Display, Push Notification, and In App Message campaign. I'm getting this error both on the google play dashboard and on crashlytics.

```
java.lang.NullPointerException: 
  at com.clevertap.clevertap_plugin.CleverTapPlugin.lambda$invokeMethodOnUiThread$0 (CleverTapPlugin.java:902)
  at com.clevertap.clevertap_plugin.-$$Lambda$CleverTapPlugin$kCoDw1W40T2vMIsGc7DuvhpNl9Y.run (-.java:6)
  at android.os.Handler.handleCallback (Handler.java:938)
  at android.os.Handler.dispatchMessage (Handler.java:99)
  at android.os.Looper.loop (Looper.java:250)
  at android.app.ActivityThread.main (ActivityThread.java:7803)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:592)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:958)
```

**Comment by darshanclevertap** [comment]
@tlworkssoftware Were you able to reproduce this on your device?

**Comment by tlworkssoftware** [comment]
> @tlworkssoftware Were you able to reproduce this on your device?

Not on my device @darshanclevertap , but Google Play reports an average of 100 errors every 2 hours. I have a base of ~200k users.

**Comment by darshanclevertap** [comment]
@tlworkssoftware We will take a further look into this and give you an update soon.

**Comment by tlworkssoftware** [comment]
@darshanclevertap any update?

**Comment by william-ct** [comment]
Hey @tlworkssoftware @macortesn , Are you guys still facing this issue in CleverTap Plugin 1.3.0?

**Comment by tlworkssoftware** [comment]
@william-ct yes, the same issue

**Comment by william-ct** [comment]
Hi @tlworkssoftware @macortesn , we've fixed this NPE in our current version of SDK(v1.4.0). Do let us know if you face any more crashes. Thanks!
